packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css

```diff
@@ -27,3 +27,13 @@
     border-color: var(--spectrum-colorwheel-border-color);
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-ColorWheel {
+    forced-color-adjust: none;
+    --spectrum-colorwheel-fill-color-disabled: GrayText;
+    --spectrum-colorwheel-border-color-disabled: GrayText;
+    --spectrum-colorwheel-border-color: ButtonText;
+    --spectrum-colorarea-border-color: ButtonText;
```

Comment:
Should the color wheel focus outline be `ButtonText` as opposed to `HighLight`, so that it is more like a Radio or Checkbox?

Reply:
There is no colorwheel focus - focus is on colorhandle